Makefile

```diff
@@ -61,6 +61,8 @@ docker-all: docker
 	$(DOCKER_BUILDX) --platform linux/ppc64le -t $(REGISTRY)/oauth2-proxy:${VERSION}-ppc64le .
 	$(DOCKER_BUILDX) --platform linux/arm/v6 -t $(REGISTRY)/oauth2-proxy:latest-armv6 .
 	$(DOCKER_BUILDX) --platform linux/arm/v6 -t $(REGISTRY)/oauth2-proxy:${VERSION}-armv6 .
+	$(DOCKER_BUILDX) --platform linux/arm64/v8 -t $(REGISTRY)/oauth2-proxy:latest-armv8 .
+	$(DOCKER_BUILDX) --platform linux/arm64/v8 -t $(REGISTRY)/oauth2-proxy:${VERSION}-armv8 .
```

Comment:
We had intended to drop the extra tags here now we are cross building. Might be worth not introducing this and instead adding to the list of DOCKER_BUILD_PLATFORM on line 42 instead? That way the main tag has all the cross compiled images for each arch and we don't need to introduce a new tag

Reply:
I'm fine with that. I would rather there be a single image anyways